### PR TITLE
test_2 added

### PR DESCRIPTION
**Pull request by TarasSkip**

### CODE REVIEW

File: pages/registration.page.js [hunk 1]
```diff
@@ -23,6 +28,30 @@ class RegistrationPage extends BasePage {
         })();
     }
 
+    async registrationWithBadEmail(bademail) { //   UNsuccessfull registration scenario
```
Comment (nbabii) [inline pages/registration.page.js]: no need for all these methods `registrationWithBadEmail, registrationWithUsedEmail, registrationWithIncorrectPassword` please update and use `registration` method instead

File: pages/registration.page.js [hunk 2]
```diff
@@ -8,13 +8,18 @@ let registrationFormNicknameLocator = by.name('login');
...
 class RegistrationPage extends BasePage {
     async waitForRegisterToBeClickable() {
         await this.getRegistrationButtonElement().waitForElementToBeClickable(100000);
     }
 
+
     // action methods
-    async registration(email, password, nickname) {
+    async registration(email, password, nickname) { //  SUCCESSFULL registration scenario
         await allure.createStep(`Step 3 - proceed with registration '${email}' '${nickname}'`, async () => {
```
Comment (nbabii) [inline pages/registration.page.js]: No need step numbers, just general info. Step number could be different

**Review by nbabii — CHANGES_REQUESTED** [review]

File: pages/registration.page.js (hunk 1 above)
Comment (TarasSkip) [reply pages/registration.page.js]: done

File: pages/registration.page.js (hunk 2 above)
Comment (TarasSkip) [reply pages/registration.page.js]: done

File: specs/test_2.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+let MainPage = require("../pages/main.page");
+let LoginPage = require("../pages/login.page");
+let RegistrationPage = require("../pages/registration.page");
+
+describe('Protractor Demo App', () => {
+    it('Registration test', async () => {
+        let rand = Math.round(Math.random() * 10000000);
+        let newemail = `vasyapupkin446+${rand}@gmail.com`;
+        //  let newpassword = 'vasyapupkin';
+        let badpassword = '1';
```
Comment (nbabii) [inline specs/test_2.js]: Please use camel case in all places like `badPassword `

File: pages/registration.page.js [hunk 4]
```diff
@@ -59,6 +70,14 @@ class RegistrationPage extends BasePage {
     getRegistrationTextElement() {
         return new TextBox(element(registrationTextLocator), "Check text");
     }
+
+    registrationFormEmailErrorTextElement() {
```
Comment (nbabii) [inline pages/registration.page.js]: `registrationFormEmailErrorTextElement` and `registrationFormPasswordErrorTextElement` used same locator, lets create one method for getting error message

**Review by nbabii — CHANGES_REQUESTED** [review]

**Pull request merged**